recipes/r-ks/build.sh

```diff
@@ -4,10 +4,5 @@
 mv DESCRIPTION DESCRIPTION.old
```

Comment:
Sorry, which two? The mv line and the grep line? And should I do that for all of my packages?
Thanks